src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs

```diff
@@ -171,6 +171,8 @@ private static bool ApplyChangesFromTC35(JObject sarifLog)
 
                     // https://github.com/oasis-tcs/sarif-spec/issues/371
                     modifiedLog |= RenameSuppressionKindValues(run);
+
+                    modifiedLog |= ConvertNotificationPhysicalLocationToLocations(run);
```

Comment:
>ConvertNotificationPhysicalLocationToLocations [](start = 35, length = 46)

Good name.

Reply:
:)

---
In reply to: [278291719](https://github.com/Microsoft/sarif-sdk/pull/1456#discussion_r278291719) [](ancestors = 278291719)